nn_sdwan/README.md

```diff
@@ -0,0 +1,61 @@
+# Agent Check: nn-sdwan
+
+## Overview
+
+This check monitors SDWAN controllers using a SDWAN platorm provided by [Netnology][1] through the Datadog Agent. Currently only Cisco vManage devices are supported.
+
```

Comment:
It would be helpful to add a bit more context here about what kinds of data is collected by this integration.